### DISCUSSION TITLE
Displaying optimization values

### DISCUSSION THREAD

**Discussion by SteveDisc** [kernc/backtesting.py#1153] [Q&A]
Hey guys, thanks in advance.

I am running a strategy and have a few variables that the script should run through in order to optimize and display the results. The normal backtesting results are printing fine, but for the life of me I cant get it to produce the values for each metric that was used to optimize the strategy.

Open to any ideas, appreciate you guys

**Comment by s-kust** [comment]
You may find thie repo - [https://github.com/s-kust/python-backtesting-template](https://github.com/s-kust/python-backtesting-template) - helpful

See Optimization of Strategy Parameters in readme
